README.md

```diff
@@ -345,7 +345,7 @@ Check [on-demand-cms-sync.yml](./.github/workflows/on-demand-cms-sync.yml) for d
 
 ### Image name
 
-The current image tag in use is: f93db8bae8be7528565c2510dbc8e3eece97983d
+The current image tag in use is: 648082468fc4b428a288daadb93b95c1f3291acf
```

Comment:
You say this, but all the workflow files still seem to be using `f93db8bae8be7528565c2510dbc8e3eece97983d`

e.g. https://github.com/yldio/asap-hub/blob/crn-1356-update-sentry-cli/.github/workflows/reusable-verify.yml#L36

Reply:
will revert